jxl/src/headers/transform_data.rs

```diff
@@ -21,9 +21,9 @@ pub struct OpsinInverseMatrix {
     #[default([11.031566901960783, -9.866943921568629, -0.16462299647058826,
                -3.254147380392157,  4.418770392156863,  -0.16462299647058826,
                -3.6588512862745097, 2.7129230470588235, 1.9459282392156863])]
-    inverse_matrix: [f32; 9],
-    #[default([0.0037930732552754493, 0.0037930732552754493, 0.0037930732552754493])]
-    opsin_biases: [f32; 3],
+    pub inverse_matrix: [f32; 9],
+    #[default([-0.0037930732552754493; 3])]
```

Comment:
Uhm, where's the sign change coming from?

Reply:
Uh, the spec says it's negative.

Reply:
Ok, so does libjxl (took way too long to figure that out...)
I guess it was just wrong before :-)